projects/workshop/desafio_rick_and_morty/README.md

```
@@ -0,0 +1,210 @@
+flutter --version
+# verifique se o Dart é >= 3.7
+
+flutter doctor -v
+# instale/ajuste o que o doctor acusar
+```
+
+---
+
+## ▶️ Rodando o projeto localmente
+
+### 1) Clonar e instalar dependências
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)